evalset/test_funcs.py

```diff
@@ -2573,6 +2575,129 @@ def do_evaluate(self, x):
         return sum(abs(x)) * prod(abs(x))
 
 
+class MixtureOfGaussians01(TestFunction):
```

Comment:
You could have called these the KevinTee functions ... or something fun.  No pressure.